pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/plugins/RabbitMQ.kt

```diff
@@ -49,10 +84,11 @@ val RabbitMQPlugin = createApplicationPlugin(
 
     val factory = pluginConfig.getConnectionFactory()
     val queueName = pluginConfig.queue
-
+    //val schemaLoader: CloudSchemaLoader = pluginConfig.createSchemaLoader()
     lateinit var connection: Connection
     lateinit var channel: Channel
-
+    val environment: ApplicationEnvironment = this@createApplicationPlugin.application.environment
+    val schemaLoader = SchemaLoaderConfiguration(environment).createSchemaLoader() // Create the schema loader here
```

Comment:
Really confused why this is mixed in with the rabbitmq plugin. Is this a mistake?

Reply:
As discussed, will be addressing it in PHDO-367